### PR TITLE
[Candidate Params] withdrawal2 not required if withdrawal1 set as null

### PR DESCRIPTION
**Pull request by stellarxo**
Bug: when a date of withdrawal has been added, it can no longer be removed. when attempting to erase and save the JS prevents save and says field is required

Now it checks if it's been reset

### CODE REVIEW

**Comment by driusan** [comment]
@jacobpenny can you review this?

**Comment by jacobpenny** [comment]
Yes I can take a look now

File: modules/candidate_parameters/jsx/consentStatus.js [hunk 1]
```diff
@@ -112,6 +112,12 @@ var ConsentStatus = React.createClass(
                     ) {
             withdrawalRequired[i] = true;
           }
+          if (this.state.formData[withdrawal] === null ||
```
Comment (stellarxo) [inline modules/candidate_parameters/jsx/consentStatus.js]: I wasn't there. Will make sure this is implemented.

Comment (stellarxo) [inline modules/candidate_parameters/jsx/consentStatus.js]: Done.

**Comment by stellarxo** [comment]
@jacobpenny can you please re-review? :D

**Comment by alisterdev** [comment]
@stellarxo Rerun ESLint plz

File: modules/candidate_parameters/js/consentStatus.js [hunk 2]
```diff
@@ -96,6 +98,9 @@ var ConsentStatus = React.createClass({
         if (this.state.formData[withdrawal] !== null && this.state.formData[withdrawal] !== undefined) {
           withdrawalRequired[i] = true;
         }
+        if (this.state.formData[withdrawal] === null || this.state.formData[withdrawal] === undefined || this.state.formData[withdrawal] === '') {
```
Comment (alisterdev) [inline modules/candidate_parameters/js/consentStatus.js]: @stellarxo This shouldn't be here, I guess you didn't recompile JSX

**Comment by jacobpenny** [comment]
It seems that already present fields aren't making their way into `formData`, so when I update they get set to null.

e.g. if I start with a form that has date of consent and date of withdrawl filled out (and saved in the db), and I update one of them, the other gets set to null (you have to refresh after the update to see).

**Comment by alisterdev** [comment]
As soon as it's rebased, it is good to go!

**Comment by driusan** [comment]
@stellarxo This still needs to be rebased. It says it can't be merged because of conflicts..

**Pull request merged**